### ISSUE TITLE
Mismatch between ip and lowered code

### ISSUE THREAD

**Issue by Keno** [JuliaDebug/Debugger.jl#228]
E.g. here:
```
In Encoder(args) at /Users/kfischer/Projects/VeronaCompiler/src/compiler.jl:215
 1  1 ─       #self#@_3 = (getfield)(#self#@_1, 1)
 2  │         ma = (getfield)(args, 1)
 3  │         mb = (getfield)(args, 2)
 4  │   @ /Users/kfischer/Projects/VeronaCompiler/src/maskedarray.jl:24 within `convert'
>5  │   %4  = (Encoder)(>=)
 6  │   %5  = (%4)(4, 2)
 7  └──       goto #3 if not %5
 8  2 ─       goto #4
 9  3 ─ %8  = (Encoder)(AssertionError)

About to run: (Encoder{typeof(>=)}(>=))(4, 2)
```
We're about to execute the code for `%5`, but the ip indicator is on `%4` (but line `5`). Presumably the code doesn't take into account the fact that lowered code can have location information printed.

**Comment by KristofferC** [comment]
What versions are you on? I thought I fixed this.

**Comment by KristofferC** [comment]
Should have been fixed in https://github.com/JuliaDebug/Debugger.jl/pull/215 which is available on Debugger 0.6.2.

If it is still not working, please comment and provide version info.
